hasher-matcher-actioner/hmalib/matchers/matchers_base.py

```diff
@@ -58,6 +58,49 @@ def get_privacy_group_matcher_active(privacy_group_id: str) -> bool:
     )
 
 
+@functools.lru_cache(maxsize=128)
```

Comment:
In both this and line 75, the maxsize value is going to cause the space used to be many times more than necessary.

Use maxsize when you want to store results for multiple arg tuples. In this case (and L75), we do not want to store results for multiple values of cache_buster.